src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/src/Model/HistoryCarbonIntensityData.cs

```diff
@@ -0,0 +1,72 @@
+using System.Text.Json.Serialization;
+
+namespace CarbonAware.DataSources.ElectricityMaps.Model;
+
+/// <summary>
+/// History Carbon Intensity collection data.
+/// </summary>
+[Serializable]
+public record HistoryCarbonIntensityData
```

Comment:
I am wondering if we should call this `RecentCarbonIntensityHistoryData` and the past range data (not part of this PR) as `PastCarbonIntensityHistoryData` to exactly mimic the Electricity Maps API spec. This way we have a clear distinction between the 2 models and is easy to decipher what each means.